### PR TITLE
Ginkgo testing

### PR DESCRIPTION
**Pull request by sjakati98**
Addresses #43

### CODE REVIEW

**Comment by sjakati98** [comment]
Addresses #43.

File: cmd/helm2go-operator-sdk/new/chart.go [hunk 1]
```diff
@@ -141,9 +141,6 @@ func (hc *HelmChartClient) DoHelmGoConversion() (*resourcecache.ResourceCache, e
 func scaffoldOverwrite(outputDir, kind, apiVersion string, rcache *resourcecache.ResourceCache) error {
 
 	ok := templating.OverwriteController(outputDir, kind, apiVersion, rcache)
```
Comment (aneeshkp) [inline cmd/helm2go-operator-sdk/new/chart.go]: What is the purpose of OK here.. how is error handled?

File: cmd/helm2go-operator-sdk/new/cmd_test.go [hunk 2]
```diff
@@ -2,122 +2,153 @@ package new
 
 import (
 	"fmt"
-	"io/ioutil"
-	"log"
 	"os"
```
Comment (aneeshkp) [inline cmd/helm2go-operator-sdk/new/cmd_test.go:5]: imports not formated well ,run go fmt

File: cmd/helm2go-operator-sdk/new/cmd_test.go [hunk 3]
```diff
@@ -2,122 +2,153 @@ package new
...
 	"path/filepath"
+
```
Comment (aneeshkp) [inline cmd/helm2go-operator-sdk/new/cmd_test.go:7]: format

File: cmd/helm2go-operator-sdk/new/cmd_test.go [hunk 4]
```diff
@@ -138,6 +169,12 @@ func createValidCommand() *cobra.Command {
 	return cmd
 }
 
+func cleanupValidOperator() {
+	gopathDir := os.Getenv("GOPATH")
+	operatorDir := filepath.Join(gopathDir, "src", "redhat-nfvpe", "helm2go-operator-sdk", "cmd", "helm2go-operator-sdk", "new", "test-operator")
```
Comment (aneeshkp) [inline cmd/helm2go-operator-sdk/new/cmd_test.go:174]: move "redhat-nfvpe" repo related to a constant so easy to refactor when moved to different repo

**Review by aneeshkp — COMMENTED** [review]
Please fix these

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**